environments/main_python_env.yaml

```diff
@@ -0,0 +1,22 @@
+name: timelapse_ViT_env
+
+channels:
+- conda-forge
+
+dependencies:
+# this is restricted to Python 3.8 at this time due to other conflicts
+- conda-forge::python=3.12
+- conda-forge::ipykernel
+- conda-forge::jupyterlab
+- conda-forge::nbconvert
+- conda-forge::pip
+- conda-forge::matplotlib
+- conda-forge::seaborn
+- conda-forge::pandas
+- conda-forge::scikit-image
+- conda-forge::scikit-learn
+- conda-forge::pytorch
+- conda-forge::torchvision
+# pip
```

Comment:
Why is pip commented-out? Or is this here to show where the pip install are?

Reply:
removed! not sure why